### PR TITLE
modify   example/reinforcement-learning/a3c/a3c.py

### PR DESCRIPTION
**Pull request by achao2013**
i try to modify a3c.py according to my understanding, the rate of convergence and score become better

### CODE REVIEW

**Comment by piiswrong** [comment]
Could you point to the equations in the paper?

File: example/reinforcement-learning/a3c/a3c.py [hunk 1]
```diff
@@ -153,8 +153,11 @@ def train():
                 module.forward(batch, is_train=True)
 
                 pi = module.get_outputs()[1]
-                h = args.beta*(mx.nd.log(pi+1e-6)+1)
-                module.backward([mx.nd.array(adv), h])
+                h = -args.beta*(mx.nd.log(pi+1e-7)*pi)
+                out_acts = np.amax(pi.asnumpy(), 1)
```
Comment (piiswrong) [inline example/reinforcement-learning/a3c/a3c.py]: I looked at the paper again and looks like should use act from https://github.com/dmlc/mxnet/pull/5473/files#diff-03ce9792a200a63897dce33fd2967c39R139 instead of amax?

Comment (achao2013) [inline example/reinforcement-learning/a3c/a3c.py]: Here 
image means the probability to act a_t in state s, i think. it means if the act is right, the probability should be big,so the loss should be low,vice versa.

Comment (achao2013) [inline example/reinforcement-learning/a3c/a3c.py]: i refer to some tensorflow implentation. Actually, if we add value loss term, the result may be better, but i think this  beyond the original paper.
